### PR TITLE
PHP 7.2 + remove XHProf/XDebug + remove vagrant + simplify docker

### PR DESCRIPTION
**Pull request by alexandresalome**

### CODE REVIEW

File: docker/php/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM php:7.2-cli
```
Comment (lolautruche) [inline docker/php/Dockerfile]: For the Dockerfile, it can be relevant to use PHP 7.4 IMHO

**Review by lolautruche — APPROVED** [review]

File: docker/php/Dockerfile (hunk 1 above)
Comment (alexandresalome) [reply docker/php/Dockerfile]: It means we would require PHP 7.4 also in the README, shouldn't we?

Comment (lolautruche) [reply docker/php/Dockerfile]: Not necessarily

Comment (alexandresalome) [reply docker/php/Dockerfile]: Fixed

**Review by lolautruche — APPROVED** [review]

**Pull request merged**